### PR TITLE
Adding Chimera Ortho Plus

### PR DESCRIPTION
**Pull request by pcoleman**
## Description
I noticed that the Chimera Ortho Plus was missing. I used Glen's initial code and updated it based off the changes in the Chimera Ortho directory on the master branch to bring it more up to date. The firmware has been tested and works appropriately.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/chimera_ortho_plus/rules.mk [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# MCU name
+MCU = atmega32u4
+
+
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/rules.mk:4]:
```suggestion
```

File: keyboards/chimera_ortho_plus/rules.mk [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+UNICODE_ENABLE = YES 		# Unicode
+# BLUETOOTH_ENABLE = yes # Enable Bluetooth with the Adafruit EZ-Key HID
+
+USB = /dev/ttyACM0
+
+OPT_DEFS += -DCHIMERA_ORTHO_PLUS_PROMICRO
+CHIMERA_ORTHO_PLUS_UPLOAD_COMMAND = while [ ! -r $(USB) ]; do sleep 1; done; \
+                         avrdude -p $(MCU) -c avr109 -U flash:w:$(TARGET).hex -P $(USB)
+
+# # project specific files
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/rules.mk:38]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
MIDI_ENABLE = no            # MIDI support
BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
AUDIO_ENABLE = no           # Audio output
FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
UNICODE_ENABLE = yes
CUSTOM_MATRIX = yes

# project specific files
```

File: keyboards/chimera_ortho_plus/readme.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+Hardware Supported: Chimera Ortho+ PCB, WaveShare core nRF51822
+
+Hardware Availability: [Gerbers](https://github.com/GlenPickle/Chimera/tree/master/ortho_plus/gerbers)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make chimera_ortho_plus:default
+
+See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
+
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/readme.md:19]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/chimera_ortho_plus/readme.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+![Chimera Ortho+](https://i.imgur.com/tuDaGdT.jpg)
+
+A split wireless 50% ortholinear keyboard 
+
+Keyboard Maintainer: [William Wilson](https://github.com/GlenPickle)  
+
+
+Hardware Supported: Chimera Ortho+ PCB, WaveShare core nRF51822
+
+Hardware Availability: [Gerbers](https://github.com/GlenPickle/Chimera/tree/master/ortho_plus/gerbers)
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/readme.md:12]:
```suggestion
* Keyboard Maintainer: [William Wilson](https://github.com/GlenPickle)
* Hardware Supported: Chimera Ortho+ PCB, WaveShare core nRF51822
* Hardware Availability: [Gerbers](https://github.com/GlenPickle/Chimera/tree/master/ortho_plus/gerbers)
```

File: keyboards/chimera_ortho_plus/config.h [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_H
+#define CONFIG_H
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/config.h:19]:
```suggestion
#pragma once
```

File: keyboards/chimera_ortho_plus/config.h [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    	UBRR1L = SERIAL_UART_UBRR; \
+    	/* baud rate */ \
+    	UBRR1H = SERIAL_UART_UBRR >> 8; \
+    	/* enable TX and RX */ \
+    	UCSR1B = _BV(TXEN1) | _BV(RXEN1); \
+    	/* 8-bit data */ \
+    	UCSR1C = _BV(UCSZ11) | _BV(UCSZ10); \
+  	} while(0)
+
+#endif
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/config.h:79]:
```suggestion
```

File: keyboards/chimera_ortho_plus/chimera_ortho_plus.h [hunk 7]
```diff
@@ -0,0 +1,80 @@
+#ifndef CHIMERA_ORTHO_PLUS_H
+#define CHIMERA_ORTHO_PLUS_H
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/chimera_ortho_plus.h:2]:
```suggestion
#pragma once
```

File: keyboards/chimera_ortho_plus/chimera_ortho_plus.h [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+) \
+{ \
+	{ KC_##k03, KC_##k40, KC_##k26, KC_##k15, KC_##k28, KC_##k01,      KC_##k42, KC_##k31, KC_##k20, KC_##k33, KC_##k08, KC_##k10 }, \
+	{ KC_##k00, KC_##k37, KC_##k14, KC_##k27, KC_##k16, KC_##k36,      KC_##k47, KC_##k19, KC_##k32, KC_##k21, KC_##k46, KC_##k11 }, \
+	{ KC_##k12, KC_##k25, KC_##k02, KC_##k39, KC_##k17, KC_##k49,      KC_##k50, KC_##k18, KC_##k44, KC_##k09, KC_##k34, KC_##k23 }, \
+	{ KC_##k24, KC_##k13, KC_##k38, KC_##k04, KC_##k05, KC_##k48,      KC_##k51, KC_##k06, KC_##k07, KC_##k45, KC_##k22, KC_##k35 }, \
+	{ KC_##k29, KC_##k41, KC_NO     KC_NO,    KC_NO,    KC_NO,         KC_##k30, KC_##k43, KC_NO,    KC_NO,    KC_NO,    KC_NO    }  \
+}
+
+#endif
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/chimera_ortho_plus.h:80]:
```suggestion
```

File: keyboards/chimera_ortho_plus/chimera_ortho_plus.c [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+}
+
+void led_set_kb(uint8_t usb_led) {
+
+}
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/chimera_ortho_plus.c:31]:
```suggestion
```

File: keyboards/chimera_ortho_plus/chimera_ortho_plus.c [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+
+void uart_init(void) {
+	SERIAL_UART_INIT();
+}
+
+void led_init(void) {
+	DDRD  |= (1<<1);
+	PORTD |= (1<<1);
+	DDRF  |= (1<<4) | (1<<5);
+	PORTF |= (1<<4) | (1<<5);
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/chimera_ortho_plus.c:11]:
```suggestion
    setPinOutput(D1);
    setPinOutput(F4);
    setPinOutput(F5);
    writePinHigh(D1);
    writePinHigh(F4);
    writePinHigh(F5);
```

File: keyboards/chimera_ortho_plus/keymaps/default/keymap.c [hunk 11]
```diff
@@ -0,0 +1,190 @@
+// this is the style you want to emulate.
+// This is the canonical layout file for the Quantum project. If you want to add another keyboard,
+
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/keymaps/default/keymap.c:3]:
```suggestion
```

File: keyboards/chimera_ortho_plus/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+#define KC_SYMB TG(_SYMBOLS)
+#define KC_SPFN LT(_NAV,KC_4)
+#define KC_SCTL MT(MOD_LCTL, KC_2)
+#define KC_SCTR MT(MOD_LCTL, KC_9)
+#define KC_SPLT MT(MOD_LALT, KC_3)
+#define KC_SPRT MT(MOD_LALT, KC_8)
+#define KC_GBRC MT(MOD_RGUI, KC_7)
+#define KC_GQOT MT(MOD_LGUI, KC_QUOT)
+#define KC_MESC LT(_MACROS, KC_ESC)
+#define KC_CAD LALT(LCTL(KC_DEL))
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/keymaps/default/keymap.c:29]:
```suggestion
#define KC_CAD LALT(LCTL(KC_DEL))
```

File: keyboards/chimera_ortho_plus/keymaps/default/keymap.c [hunk 13]
```diff
@@ -0,0 +1,190 @@
...
+  switch(keycode) {
+    /* include some kind of library or header */
+    case KC_INCL:
+      if (record->event.pressed) {
+        SEND_STRING("#include <>" SS_TAP(X_LEFT));
+      }
+      return false;
+    case KC_PULL:
+      if (record->event.pressed) {
+        SEND_STRING("git pull" SS_TAP(X_ENTER));
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/keymaps/default/keymap.c:137]:
```suggestion
        SEND_STRING("git pull\n");
```

File: keyboards/chimera_ortho_plus/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,190 @@
...
+      }
+      return false;
+    case KC_PUSH:
+      if (record->event.pressed){
+        SEND_STRING("git push" SS_TAP(X_ENTER));
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/keymaps/default/keymap.c:142]:
```suggestion
        SEND_STRING("git push\n");
```

File: keyboards/chimera_ortho_plus/keymaps/default/keymap.c [hunk 15]
```diff
@@ -0,0 +1,190 @@
...
+      }
+      return false;
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/keymaps/default/keymap.c:144]:
```suggestion
      return false;
```

File: keyboards/chimera_ortho_plus/keymaps/default/keymap.c [hunk 16]
```diff
@@ -0,0 +1,190 @@
...
+    case KC_PUSH:
+      if (record->event.pressed){
+        SEND_STRING("git push" SS_TAP(X_ENTER));
+      }
+      return false;    
+	case KC_SCAP:
+      if (record->event.pressed){
+        layer_on(_CAPS);
+        register_code(KC_CAPSLOCK);
+        unregister_code(KC_CAPSLOCK);
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/keymaps/default/keymap.c:149]:
```suggestion
        tap_code(KC_CAPS);
```

File: keyboards/chimera_ortho_plus/keymaps/default/keymap.c [hunk 17]
```diff
@@ -0,0 +1,190 @@
...
+      }
+      return false;
+    case KC_SCOF:
+      if (record->event.pressed){
+        layer_off(_CAPS);
+        register_code(KC_CAPSLOCK);
+        unregister_code(KC_CAPSLOCK);
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/keymaps/default/keymap.c:156]:
```suggestion
        tap_code(KC_CAPS);
```

**Comment by pcoleman** [comment]
Thanks for the review. The changes make sense.

File: keyboards/chimera_ortho_plus/chimera_ortho_plus.h [hunk 18]
```diff
@@ -0,0 +1,77 @@
...
+  k36, k37, k38, k39, k40, k41,      k42, k43, k44, k45, k46, k47, \
+                      k48, k49,      k50, k51 \
+) \
+{ \
+	{ KC_##k03, KC_##k40, KC_##k26, KC_##k15, KC_##k28, KC_##k01,      KC_##k42, KC_##k31, KC_##k20, KC_##k33, KC_##k08, KC_##k10 }, \
+	{ KC_##k00, KC_##k37, KC_##k14, KC_##k27, KC_##k16, KC_##k36,      KC_##k47, KC_##k19, KC_##k32, KC_##k21, KC_##k46, KC_##k11 }, \
+	{ KC_##k12, KC_##k25, KC_##k02, KC_##k39, KC_##k17, KC_##k49,      KC_##k50, KC_##k18, KC_##k44, KC_##k09, KC_##k34, KC_##k23 }, \
+	{ KC_##k24, KC_##k13, KC_##k38, KC_##k04, KC_##k05, KC_##k48,      KC_##k51, KC_##k06, KC_##k07, KC_##k45, KC_##k22, KC_##k35 }, \
+	{ KC_##k29, KC_##k41, KC_NO     KC_NO,    KC_NO,    KC_NO,         KC_##k30, KC_##k43, KC_NO,    KC_NO,    KC_NO,    KC_NO    }  \
+}
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/chimera_ortho_plus.h:77]:
```suggestion
```

**Comment by pcoleman** [comment]
Let me know if any other changes need to be made

File: keyboards/chimera_ortho_plus/chimera_ortho_plus.h [hunk 19]
```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "quantum.h"
+
+#define red_led_off   PORTF |= (1<<5)
+#define red_led_on    PORTF &= ~(1<<5)
+#define blu_led_off   PORTF |= (1<<4)
+#define blu_led_on    PORTF &= ~(1<<4)
+#define grn_led_off   PORTD |= (1<<1)
+#define grn_led_on    PORTD &= ~(1<<1)
```
Comment (noroadsleft) [inline keyboards/chimera_ortho_plus/chimera_ortho_plus.h:10]: This block *could* use the [GPIO Control functions](https://docs.qmk.fm/#/internals_gpio_control?id=gpio-control-idgpio-control), but I'm not gonna call it required.

File: keyboards/chimera_ortho_plus/chimera_ortho_plus.h [hunk 20]
```diff
@@ -0,0 +1,62 @@
...
+#define set_led_off     PORTF &= ~(1<<LED_B) & ~(1<<LED_R) & ~(1<<LED_G)
+#define set_led_red     PORTF = PORTF & ~(1<<LED_B) & ~(1<<LED_G) | (1<<LED_R)
+#define set_led_blue    PORTF = PORTF & ~(1<<LED_G) & ~(1<<LED_R) | (1<<LED_B)
+#define set_led_green   PORTF = PORTF & ~(1<<LED_B) & ~(1<<LED_R) | (1<<LED_G)
+#define set_led_yellow  PORTF = PORTF & ~(1<<LED_B) | (1<<LED_R) | (1<<LED_G)
+#define set_led_magenta PORTF = PORTF & ~(1<<LED_G) | (1<<LED_R) | (1<<LED_B)
+#define set_led_cyan    PORTF = PORTF & ~(1<<LED_R) | (1<<LED_B) | (1<<LED_G)
+#define set_led_white   PORTF |= (1<<LED_B) | (1<<LED_R) | (1<<LED_G)
+*/
+
```
Comment (noroadsleft) [inline keyboards/chimera_ortho_plus/chimera_ortho_plus.h:44]:
```suggestion
```

File: keyboards/chimera_ortho_plus/info.json [hunk 21]
```diff
@@ -0,0 +1,65 @@
+{
+  "keyboard_name": "Chimera Ortho Plus",
+  "keyboard_folder": "chimera_ortho_plus", 
+  "maintainer": "qmk",
+  "width": 13.5,
+  "height": 6,
+  "layouts": {
+    "LAYOUT": {
+      "layout": [
+	    {"label": "{", "x": 0, "y": 0},
```
Comment (noroadsleft) [inline keyboards/chimera_ortho_plus/info.json:10]: Alignment fix.

```suggestion
        {"label": "{", "x": 0, "y": 0},
```

File: keyboards/chimera_ortho_plus/readme.md [hunk 22]
```diff
@@ -0,0 +1,15 @@
+# Chimera Ortho+
+
+![Chimera Ortho+](https://i.imgur.com/tuDaGdT.jpg)
```
Comment (noroadsleft) [inline keyboards/chimera_ortho_plus/readme.md:3]: Swaps this image for a version with fewer pixels.

```suggestion
![Chimera Ortho+](https://i.imgur.com/tuDaGdTl.jpg)
```

File: keyboards/chimera_ortho_plus/readme.md [hunk 23]
```diff
@@ -0,0 +1,15 @@
...
+
+A split wireless 50% ortholinear keyboard
```
Comment (noroadsleft) [inline keyboards/chimera_ortho_plus/readme.md:5]:
```suggestion
A split wireless 50% ortholinear keyboard.
```

File: keyboards/chimera_ortho_plus/keymaps/default/keymap.c [hunk 24]
```diff
@@ -0,0 +1,185 @@
...
+        tap_code(KC_CAPS);
+      }
+      return false;
+  }
+  return true;
+};
+ 
+
+void matrix_scan_user(void) {
+    uint8_t layer = biton32(layer_state);
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/keymaps/default/keymap.c:160]:
```suggestion
    uint8_t layer = get_highest_layer(layer_state);
```

File: keyboards/chimera_ortho_plus/config.h [hunk 25]
```diff
@@ -0,0 +1,76 @@
...
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
```
Comment (fauxpark) [inline keyboards/chimera_ortho_plus/config.h:25]:
```suggestion
#define PRODUCT_ID      0xDE1D
```
```
echo -n "Chimera Ortho Plus"|shasum
```

**Comment by zvecr** [comment]
Thanks!

**Pull request merged**